src/zrc/api/serializers.py

```diff
@@ -79,7 +80,7 @@ class Meta:
             },
             'zaaktype': {
                 # TODO: does order matter here with the default validators?
-                'validators': [URLValidator()],
+                'validators': [URLValidator(headers=ztc_auth.credentials)],
```

Comment:
Ja maar...

Nu heb je je auth vastgelegd op een bepaald ZTC, en je weet niet zeker of deze URL naar die ZTC wijst

Reply:
He, heb je 't toch gezien :-)

Dit wil ik inderdaad slimmer gaan maken - was een eerste proof of concept met statische auth. Denk dat het prima mogelijk is om de validator de value mee te geven als context/argument voor de callable, zodat die dynamisch een lookup kan doen van de creds.

Reply:
Zie #26